README.md

```diff
@@ -103,7 +105,8 @@ m5
                     ├── tlf-efficacy.R
                     ├── tlf-kmplot.R
                     ├── tlf-primary.R
-                    └── renv.lock
+                    ├── pilot3utils_0.0.2.zip
```

Comment:
Should we change the zip file name as "." is not permitted per FDA response letter?
```suggestion
                    ├── pilot3utils_0_0_2.zip
```
![image](https://github.com/user-attachments/assets/2dde758f-2091-4d75-af7f-a13b03bfc104)